{"name": "risk user -- @1.4 cases", "status": "passed", "steps": [{"name": "Given risk makes get request: /reconciliation/all_users_data/xlsx", "status": "passed", "start": 1610701700504, "stop": 1610701701103}, {"name": "Then check response of request: You do not have permission to perform this action. is False", "status": "passed", "start": 1610701701104, "stop": 1610701701278}], "parameters": [{"name": "url", "value": "/reconciliation/all_users_data/xlsx"}, {"name": "method", "value": "get"}, {"name": "bool", "value": "False"}], "start": 1610701700500, "stop": 1610701701278, "uuid": "7357437c-8501-4dcc-bb1e-e4beac53231e", "historyId": "ad308c72d421c1e6a04fe8711c9cc284", "fullName": "features/III-reconciliation_access.feature:risk user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}